sips/sip-60.md

```diff
@@ -54,24 +55,68 @@ TBD: Should the escrowed SNX be liquidated at a larger discount? or vested and t
 ```
 interface ISynthetixEscrow {
     // Views
-    
+
+    // Check whether an address has migrated from the Old reward escrow to the new contract
+    function accountMigrated(address account) external view returns (bool);
```

Comment:
is this necessary you think? Basically if they have any escrow entries then they've been migrated...

Reply:
- Only on the condition that Claiming Rewards is blocked if there isn't any existing entries on the new reward escrow, so that no escrow entries are added before they migrate.  

- The code has to check whether the address has a `totalEscrowedAccountBalance` on the new contract and also missing the escrow entries.